build.gradle

```diff
@@ -16,13 +16,16 @@ buildscript {
     repositories {
         google()
         jcenter()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.2'
+        classpath 'com.android.tools.build:gradle:4.2.1'
```

Comment:
Due to this gradle build tool plugin the android build tools sdk will also have to be updated to `30.0.2` for the build to pass ,modify its references in travis and the build.gradle.

Reply:
@odaridavid Alright I will fix it 👍